src/api/auth/src/authentication.js

```diff
@@ -107,17 +107,43 @@ const strategy = new SamlStrategy(
      *   "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress": "first.last@email.com"
      *   "email": "first.last@email.com"
      * }
+     *
+     * We only really care about the emailaddress claim, which we also use in the Users service.
      */
+    const email =
+      senecaProfile['http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress'];
+
+    async function lookupTelescopeUser(id) {
```

Comment:
I assume that when this function is invoked, we would have a check to see if the User creating the query has permissions to do so, right?